frontend/src/modules/Shares/views/ShareView.js

```diff
@@ -489,33 +487,11 @@ export function SharedItem(props) {
         )}
       </TableCell>
       <TableCell>
-        <div style={{ display: 'flex', alignItems: 'left' }}>
-          {item.healthStatus === 'Unhealthy' ? (
-            <Tooltip title={<Typography>{item.healthStatus}</Typography>}>
-              <GppBadIcon color={'error'} />
-            </Tooltip>
-          ) : item.healthStatus === 'Healthy' ? (
-            <Tooltip title={<Typography>{item.healthStatus}</Typography>}>
-              <VerifiedUserIcon color={'success'} />
-            </Tooltip>
-          ) : (
-            <Tooltip
-              title={
-                <Typography>{item.healthStatus || 'Undefined'}</Typography>
-              }
-            >
-              <PendingIcon color={'info'} />
-            </Tooltip>
-          )}
```

Comment:
nit: add some space between the tag and tooltip and the text with the verification time. At the moment it is really tight
![image](https://github.com/data-dot-all/dataall/assets/71252798/526abc9d-e8ad-4c0f-a427-0ff5e8a72e76)